site/e2e/tests/login.spec.ts

```diff
@@ -10,7 +10,7 @@ test("Login takes user to /workspaces", async ({ baseURL, page }) => {
   const signInPage = new SignInPage(baseURL, page)
   await signInPage.submitBuiltInAuthentication(email, password)
 
-  const workspacesPage = new WorkspacesPage(baseURL, page)
+  const workspacesPage = new WorkspacesPage(baseURL, page, "?filter=owner%3Ame")
```

Comment:
@jsjoeio I realized after we stopped pairing that the old behavior was to display only the users' workspaces on first load, which I think is valuable. So instead of stripping off any query params as we discussed, I 1) fixed the workspaces page so the `filter` param is no longer empty and 2) I added an option to pass params in the test scaffolding, then updated that specific test so we are testing for those params in that test only. I think that's the right path forward, but let me know if you disagree!
![Screen Shot 2022-06-29 at 12 15 56 PM](https://user-images.githubusercontent.com/19142439/176490754-698cae50-86f4-4a23-aed0-1c99c39db4dc.png)

Reply:
Hooray! Nice realization 🌮 That seems right to me.